pkgs/by-name/un/unfloader/package.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  ncurses,
+  libftdi1,
+  libusb1,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "UNFLoader";
```

```suggestion
  pname = "unfloader";
```

[Guidelines request packages name to be all lowercase.](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming)

> It must not contain uppercase letters.